src/pages/Login.tsx

```diff
@@ -1,46 +1,84 @@
 import { useState } from "react";
 import { useNavigate } from "react-router-dom";
-import KakaoLogin from "../components/KakaoLogin";
+import KakaoLogin from "../components/Login/KakaoLogin";
 
+type stateType = {
+  email: string;
+  password: string;
+};
 
 export default function Login() {
-  const [email, setEmail] = useState('');
-  const [password, setPassword] = useState('');
+  const [state, setState] = useState<stateType>({
+    email: "",
+    password: "",
+  });
   const navigate = useNavigate();
 
   const onChange = (event: React.ChangeEvent<HTMLInputElement>) => {
-    const { target: { name, value } } = event;
-    if (name === 'email') {
-      setEmail(value)
-    } else if (name === 'password') {
-      setPassword(value);
+    const {
+      target: { name, value },
+    } = event;
+    if (name === "email") {
+      setState({
+        ...state,
+        email: value,
+      });
+    } else if (name === "password") {
+      setState({
+        ...state,
+        password: value,
+      });
```

Comment:
이 부분은 이런식으로 변경하는게 어떨까요?
```suggestion
setState((values) => ({...values, [name]: value}))
```

Reply:
넵 좋습니다.